www/assets/css/theme.css

```diff
@@ -270,17 +263,14 @@ header nav li {
   margin: 0;
 }
 
-header nav li:has(.active) > a {
+header nav li:has(.active) a {
   background-color: var(--bg-main);
   color: var(--fg-banner);
 }
 
-header nav li {
-  border-left: 1px solid var(--fg-banner-border);
-}
-
-header nav li:last-child {
-  border-right: 1px solid var(--fg-banner-border);
+/* submenus */
+header nav ul ul {
+  display: none;
```

Comment:
That’s kind of sad… The default render is awful but maybe we could fix it by rendering those child elements in a separate `div` right below the nav bar, though keeping the vertical alignment could be tricky.

One *very dirty* hack we could use would be to have *both* the current style and the « other div » style in the markup, but only render one of those depending on screen width…

Reply:
Right now we’re limited by the markup itself, which relies on nested lists : 
```html
<nav>
  <ol>
    <li> home </li>
    <li> section
      <ol>
        <li> subitem 1 </li>
        <li> subitem 2 </li>
      </ol>
    </li>
    <li> about </li>
  </ol>
</nav>
```

The current `position: absolute;` hack allows some kind of sub-menu in desktop mode (>768px), though it’s ugly at the moment : 
![image](https://github.com/Nuclear-Squid/ergol/assets/457778/40ee2962-868f-4200-a899-3f4db4db0411)

I suspect we could have something much more flexible with this kind of nested lists, but we’d have to make sure it’s valid and accessible : 
```html
<nav>
  <ol>
    <li> home </li>
    <li> section </li>
    <ol>
      <li> subitem 1 </li>
      <li> subitem 2 </li>
    </ol>
    <li> about </li>
  </ol>
</nav>
```

And more practically, I think the general solution is to have a vertical menu in most cases : 
- in wide mode (>1200px ?) this menu could be in the left margin
- in narrow mode (<768px ?) this should be a hamburger menu

Long story short, I’m not pleased with the current solution either. It’s not *good* yet, but I think it’s *better* than what we have and a sane basis to explore from.